app/src/main/java/at/roteskreuz/stopcorona/model/managers/DatabaseCleanupManager.kt

```diff
@@ -59,8 +54,11 @@ class DatabaseCleanupManagerImpl(
         )
     }
 
-    override suspend fun removeSentTemporaryExposureKeys() {
-        temporaryExposureKeysDao.removeSentTemporaryExposureKeys()
+    override suspend fun removeSentYellowTemporaryExposureKeys() {
+        temporaryExposureKeysDao.removeSentInfectionMessagesOlderThan(
+            MessageType.InfectionLevel.Yellow,
+            ZonedDateTime.now().toRollingStartIntervalNumber()
+        )
     }
 
     private fun cleanupInfectionMessages() {
```

Comment:
there is no infection messages anymore, why is this not all renamed to cleanupTemporaryExposureKeys

Reply:
There are still received infections messages from the old version which will be replaced in the new download logic. At the moment the name is still valid, but not complete, a complete name for this point could be cleanupInfectionMessagesAndSentTemporaryExposureKeys(). 

But I think a temporary name is not necessary, it can be updated appropriately when the part for the download is updated.

Reply:
okay!